tests/test_metric.py

```diff
@@ -469,3 +473,11 @@ def test_input_tf(self):
             metric.add(prediction=pred, reference=ref)
         self.assertDictEqual(expected_results, metric.compute())
         del metric
+
+
+def test_seqeval_raises_when_incorrect_scheme():
```

Comment:
Could you move this to test_metric_common.py ?

Reply:
Done, thank you